cicd/crabserver_pypi/manage

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+##H Usage: manage ACTION [ATTRIBUTE] [SECURITY-STRING]
+##H
```

Comment:
for example, here I would add a pointer tohttps://github.com/dmwm/deployment/blob/master/crabserver/manage